integrations/datadog/client.py

```diff
@@ -0,0 +1,264 @@
+import json
+from typing import Any, AsyncGenerator, Optional
+from urllib.parse import urlparse, urlunparse
+import httpx
+
+from loguru import logger
+from port_ocean.context.event import event
+from port_ocean.utils import http_async_client
+
+MAX_PAGE_SIZE = 50
```

Comment:
Increase PAGE SIZE to 100 to overcome potential rate limit (please check with their API reference if there is a hard limit on the page size param)